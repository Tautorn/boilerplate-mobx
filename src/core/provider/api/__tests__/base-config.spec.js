import { getBaseConfig } from '../base-config'

describe('Base Config', () => {

	const config = getBaseConfig()

	it('should return base configuration', () => {
		expect(config).toHaveProperty('baseURL')
		expect(config).toHaveProperty('headers')
		expect(config.headers).toHaveProperty('Accept')

		expect(config).toMatchSnapshot()

	})

	it('should return header like application/json', () => {
		expect(config.headers.Accept).toEqual('application/json')
	})
})
