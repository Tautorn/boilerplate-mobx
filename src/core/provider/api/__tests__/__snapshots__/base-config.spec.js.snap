// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base Config should return base configuration 1`] = `
Object {
  "baseURL": "http://localhost:3000",
  "headers": Object {
    "Accept": "application/json",
    "Accept-Language": "en-US",
  },
}
`;
